data/json/items/ammo/38.json

```diff
@@ -65,5 +65,15 @@
     "proportional": { "price": 0.7, "damage": 0.9, "dispersion": 1.1 },
     "extend": { "effects": [ "RECYCLED" ] },
     "delete": { "effects": [ "NEVER_MISFIRES" ] }
+  },
+  {
+    "id": "bp_38_special",
+    "copy-from": "38_special",
+    "type": "AMMO",
+    "name": "blackpowder .38 Special",
+	"description": "This .38 Special cartridge is loaded with a 200gr cast lead hollowpoint and 23gr of blackpowder. Much slower than regular .38s, but tremendous stopping power.",
```

Comment:
```suggestion
	"description": "This .38 Special cartridge is loaded with a 200gr cast lead hollowpoint and 23gr of blackpowder. Much slower than regular .38s, but has tremendous stopping power.",
```

Reply:
I disagree. It's a sentence fragment and I don't feel that adding "has" would improve clarity at all.

Reply:
Even if it doesn't improve clarity, adding has makes it flow better (maybe `it has` would be better).